src/main/resources/assets/bingolobby/lang/pl_pl.json

```diff
@@ -0,0 +1,24 @@
+{
+  "bingolobby.command.disabled": "Ta komenda jest wyłączona. Zamiast niej skorzystaj z lobby.",
+  "bingolobby.nojoin.novip": "To jest drużyna VIP, a Ty nie jesteś VIP-em.",
+  "bingolobby.nojoin.full": "Ta drużyna jest pełna.",
+  "bingolobby.command.vipadd": "Następująca liczba graczy została VIP-ami: %s.",
+  "bingolobby.command.vipremove": "Usunięto status VIP od następującej liczby graczy: %s.",
+  "bingolobby.command.vipteamadd": "Drużyna %s jest teraz drużyną VIP.",
+  "bingolobby.command.vipteamremove": "Drużyna %s nie jest już drużyną VIP.",
+  "bingolobby.command.maxplayers.limited": "Maksymalna liczba graczy na drużynę jest teraz ustawiona na: %s.",
+  "bingolobby.command.maxplayers.unlimited": "Maksymalna liczba graczy na drużynę jest teraz ustawiona na: Bez limitu.",
+  "bingolobby.nojoin.noactive": "Nie możesz dołączyć do drużyny, gdy nie ma utworzonej żadnej gry.",
+  "bingolobby.countdown.seconds": "Gra rozpocznie się za %s sek.",
+  "bingolobby.countdown.minutes": "Gra rozpocznie się za %s min.",
+  "bingolobby.countdown.set": "Rozpoczęto odliczanie %s min i %s sek.",
+  "bingolobby.scoreboard.online": "Graczy online: %s",
+  "bingolobby.scoreboard.no_joining": "Drużyny będą wkrótce otwarte.",
+  "bingolobby.scoreboard.players_per_team": "Graczy na drużynę: %s",
+  "bingolobby.scoreboard.status.info": "Status: %s",
+  "bingolobby.scoreboard.status.player": "Gracz",
+  "bingolobby.scoreboard.status.vip": "VIP",
+  "bingolobby.scoreboard.noteam": "Brak drużyny",
+  "bingolobby.scoreboard.nogame": "Brak utworzonej gry",
```

Comment:
I think that this should be changed to "No created game" in the U.S. English translation file as the word "active" is not used in any Bongo command and may be confusing.